config/notifiers.go

```diff
@@ -753,6 +753,7 @@ type TelegramConfig struct {
 	BotToken             Secret `yaml:"bot_token,omitempty" json:"token,omitempty"`
 	BotTokenFile         string `yaml:"bot_token_file,omitempty" json:"token_file,omitempty"`
 	ChatID               int64  `yaml:"chat_id,omitempty" json:"chat,omitempty"`
+	MessageThreadID      int64  `yaml:"message_thread_id,omitempty" json:"message_thread_id,omitempty"`
```

Comment:
according to the [documentation](https://core.telegram.org/bots/api#sendmessage) the field is Integer, which is a 32-bit signed integer.

Reply:
> according to the [documentation](https://core.telegram.org/bots/api#sendmessage) the field is Integer, which is a 32-bit signed integer. Perhaps let's change it to int32?

Telegram doesn't use 32 bit integer anymore since [API Layer 133](https://core.telegram.org/api/layers#layer-133), Integer in Telegram documentation can be safely assumed as 64-bit integer

Reply:
I updated MessageThreadID as int32, and also updated my branch with the upstream one.